tests/tkupchyn/homework_08/test_warior.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from tkupchyn.homework_08.warior import (Warrior, Knight, fight)
+
+
+@pytest.mark.parametrize('unit1, unit2, expected_result',
+                         (
+                                 (Warrior(), Warrior(), True),
+                                 (Warrior(), Knight(), False),
+                                 (Warrior(), Warrior(), True),
+                                 (Knight(), Knight(), True),
+
+                         ))
+def test_fight(unit1, unit2, expected_result):
+    assert fight(unit1, unit2) == expected_result
+
+
+@pytest.mark.parametrize('unit1, unit2, expected_result',
+                         (
+                                 (Warrior(), Warrior(),
+                                  {'before_fight': True, 'after_fight': True}),
```

Comment:
by such identation you waste much space which can be used in case if parametrise is more comples and takes more space